node/cli/src/service.rs

```diff
@@ -359,13 +359,18 @@ mod tests {
 		service_test::sync(
 			chain_spec::integration_test_config(),
 			|config| new_full(config),
-			|config| new_light(config),
+			|mut config| {
+				// light nodes are unsupported
+				config.roles = Roles::FULL;
+				new_full(config)
+			},
 			block_factory,
 			extrinsic_factory,
 		);
 	}
 
 	#[test]
+	#[cfg(any())]
```

Comment:
Why is this needed?

Reply:
dunno, must have come in with one of the merges. will remove